src/tools/qt-gui/qml/GUIActions.qml

```diff
@@ -360,13 +376,15 @@ Item {
 		id: pluginInfoAction
 		text: qsTr("Show &Plugin Info ...")
 		iconSource: "icons/help-about.png"
+		iconName: (guiSettings.useSystemIconTheme) ? "help-about" : ""
 		onTriggered: pluginInfo.show()
 	}
 
 	Action {
 		id: quitAction
 		text: qsTr("Quit")
 		iconSource: "icons/application-exit.png"
+		iconName: (guiSettings.useSystemIconTheme) ? "application-exit" : ""
```

Comment:
what about writing a little function for this? Doesn't `useIcon("application-exit")` look nicer?